### PR TITLE
Rename all cluster variables from hosts to nodes

### PR DESCRIPTION
**Pull request by ericzolf**
### What does this PR do?
It appears that tower_hosts clashes with tower_collection/roles/hosts' variable of same name;
and the other nodes (AH and database) for being consistent.

### How should this be tested?
Inventory variables must be renamed and provisioning tried.

### Is there a relevant Issue open for this?
see Google Chat...

### Other Relevant info, PRs, etc.
n/a

### CODE REVIEW

**Review by Tompage1994 — APPROVED** [review]
I'm happy with this as a change. We need to be careful that the change in variable name is explained (possibly add something to the readme?).

Other than that I'm happy. Will wait for @sean-m-sullivan to approve also

File: README.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ collections:
 
 ## Release and Upgrade Notes
 
+- After v0.4.0 the variable `tower_hosts` was renamed to `tower_nodes` in order to not collide with the variable of the same name in the redhat\_cop.tower\_configuration collection.
```
Comment (Tompage1994) [inline README.md:41]: `- After v1.0.0 the variable `tower_hosts` was renamed to `tower_nodes` in order to not collide with the variable of the same name in the redhat\_cop.tower\_configuration collection.`

**Review by Tompage1994 — CHANGES_REQUESTED** [review]
We are on v1.0.0 right now so if you could change the readme that would be great

**Review by sean-m-sullivan — APPROVED** [review]
Looks good to me, like the idea of nodes and haven't thought of it before. This will need rewritten to work with the new installer as well when it becomes official, I believe I have the inventory file for it though.

File: README.md (hunk 1 above)
Comment (sean-m-sullivan) [reply README.md:41]: This need the version changed still to 1.0.0

**Review by Tompage1994 — APPROVED** [review]

**Pull request merged**